redfish-core/lib/fan.hpp

```diff
@@ -562,8 +562,6 @@ inline void getValidFan(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
                     }
                     if (chassisName != chassisId)
                     {
-                        BMCWEB_LOG_ERROR << "The fan obtained at this time "
```

Comment:
I am good removing these because we have multiple chassis and these would log all the time.